flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
that's disappointing

Reply:
Yes, ARM builds will be possible once https://github.com/G-Ray/transmission-native/issues/20 is resolved :rocket:

Reply:
you could build it here instead.

Reply:
I would like, I hope I will be able to resolve the issues I described in the PR comment in the near future, to build from sources :crossed_fingers: